final/recommendations/alternate-eval.tex

```diff
@@ -13,15 +14,3 @@ \subsection{Adding support for alternative evaluation strategies}
 \texttt{``IEvaluationStrategy''}. The language designer can register alternative
 evaluation strategies with the REPL by extending the Guice module of the backend
 and overriding the \texttt{bindEvalStrategies} method.
-
-Finally, the evaluation strategy that will be used to evaluate input terms can
```

Comment:
Instead of removing this, you should add a reference to the section you're writing.

> Finally, the evaluation strategy that will be used to evaluate input terms can be configured via the ShellFacet ESV extensions, see \cref{...}.